crates/tree-sitter-essence/grammar.js

```diff
@@ -33,13 +33,18 @@ module.exports = grammar ({
 
     FALSE: $ => "false",
 
-    variable: $ => /[a-zA-Z_][a-zA-Z0-9_]*/,
+    find_kw: $ => "find",
+
+
+    // Disallow reserved keywords from being recognized as variables anywhere
+    // Keep precedence low so true keywords still win where both could match
+    variable: $ =>(/[a-zA-Z_][a-zA-Z0-9_]*/),
```

Comment:
I don't get this - how does it avoid getting `find bool : bool` parsed with the first bool as the variable name?